src/rpc/blockchain.cpp

```diff
@@ -563,6 +574,12 @@ UniValue getblockheader(const JSONRPCRequest& request)
             "  \"mediantime\" : ttt,    (numeric) The median block time in seconds since epoch (Jan 1 1970 GMT)\n"
             "  \"nonce\" : n,           (numeric) The nonce\n"
             "  \"bits\" : \"1d00ffff\", (string) The bits\n"
+            "  \"shielded_pool_value\": (object) Block shielded pool value\n"
+            "  {\n"
+            "     \"chainValue\":        (bool) Total value held by the Sapling circuit up to and including this block\n"
+            "     \"valueDelta\":        (bool) Change in value held by the Sapling circuit over this block\n"
```

Comment:
(e971fccc144386e16be0cc7ebf2ecbfcab0dc5e8)
numeric, not bool. Value expressed in PIV.